playbooks/adhoc/openshift_hosted_logging_efk.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 - hosts: masters[0]
   roles:
-  - role: openshift_hosted_logging
+  - role: openshift_logging
```

Comment:
i dont think you meant to modify this file yet, not until QE has certified both new roles